ingestors/kafka/src/main/scala/hydra/kafka/programs/CreateTopicProgram.scala

```diff
@@ -100,7 +101,8 @@ final class CreateTopicProgram[F[_]: Bracket[*[_], Throwable]: Sleep: Logger](
 
   private def publishMetadata(
       topicName: Subject,
-      createTopicRequest: TopicMetadataV2Request
+      createTopicRequest: TopicMetadataV2Request,
+      headers: Option[Headers]
```

Comment:
Since this will always be `None` it would probably make sense to remove it as a param here and just pass `None` directly for now.